tests/gui/test_commands.py

```
@@ -1046,3 +1048,65 @@ def on_triggered():
     diff = np.nan_to_num(new_inst.numpy() - copy_instance.numpy(), nan=offset)
     assert np.all(diff == offset)
+
+
+def test_go_next_instance_change(centered_pair_predictions: Labels):
+    """Test that goto next instance change command works correctly."""
+    # Set up test data
+    labels = centered_pair_predictions
+    video = labels.videos[0]
+
+    # Create frames with different numbers of instances
+    lf1 = context.labels.find(video, 0, return_new=True)[
```

Ruff (0.8.2)

Undefined name `context`

(F821)